src/server/enrollment-system.js

```diff
@@ -263,12 +98,12 @@ function spanishHispanicToSDSCode(isSpanishHispanicLatino) {
 function veteranToRaces(veteran) {
   // from VHA Standard Data Service (ADRDEV01) HL7 24 Race Map List
   const races = [];
-  if (veteran.isAmericanIndianOrAlaskanNative) races.push({ race: '1002-5' });
-  if (veteran.isAsian) races.push({ race: '2028-9' });
-  if (veteran.isBlackOrAfricanAmerican) races.push({ race: '2054-5' });
-  if (veteran.isNativeHawaiianOrOtherPacificIslander) races.push({ race: '2076-8' });
-  if (veteran.isWhite) races.push({ race: '2106-3' });
-  return races;
+  if (veteran.isAmericanIndianOrAlaskanNative) races.push('1002-5');
+  if (veteran.isAsian) races.push('2028-9');
+  if (veteran.isBlackOrAfricanAmerican) races.push('2054-5');
+  if (veteran.isNativeHawaiianOrOtherPacificIslander) races.push('2076-8');
+  if (veteran.isWhite) races.push('2106-3');
```

Comment:
yup.

I couldn't find a spec or anything for the format, but it generates sensible looking xml when I capture it.